### PR TITLE
tests: drivers: build_all: Add test for video drivers

### PR DESCRIPTION
**Pull request by galak**

### CODE REVIEW

**Review by MaureenHelm — DISMISSED** [review]

**Comment by galak** [comment]
Depends on  #47169 and  #47168

**Comment by galak** [comment]
@MaureenHelm can I get you to re-review this

**Comment by galak** [comment]
@loicpoulain can I get you to take a look

**Review by MaureenHelm — DISMISSED** [review]

File: tests/drivers/build_all/video/src/main.c [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
+ *
```
Comment (mmahadevan108) [inline tests/drivers/build_all/video/src/main.c:3]: Please fix the Copyright Year

File: tests/drivers/build_all/video/app.overlay [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2020, Linaro Ltd.
```
Comment (mmahadevan108) [inline tests/drivers/build_all/video/app.overlay:2]: Please update the Copyright Year

**Review by MaureenHelm — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**